Buzzer/Buzzer_function/Buzzer_function.ino

```diff
@@ -1,11 +1,12 @@
-buzzer_pin = 8;
-void setup (){}
 
-void loop (){
-buzzer();
-}
+//void setup(){} 
+
+//void loop(){Buzzer();}
 
-//Function is used at the end to make the Buzzer active
-void buzzer () {
-tone (buzzer_pin, 262);
+void Buzzer()
+{
+  tone (6, 262);
+  delay (4000);
```

Comment:
I was thinking we stopped all transmission in the final state.
But we actually should not.

Ok, Make it have a duration.
Duration can be 5sec.